components/Pages/Swap/hooks/useTransaction.tsx

```diff
@@ -56,19 +56,19 @@ type Params = {
 }
 
 export const useTransaction = ({
-  enabled,
-  swapAddress,
-  swapAssets,
-  client,
-  senderAddress,
-  msgs,
-  encodedMsgs,
-  amount,
-  price,
-  onBroadcasting,
-  onSuccess,
-  onError,
-}: Params) => {
+                                 enabled,
```

Comment:
This formatting looks a little off, do you have like 8 tabs as your setting ? Maybe fine